bitnami/contour/Chart.yaml

```diff
@@ -25,4 +25,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/containers/tree/main/bitnami/contour
   - https://projectcontour.io
-version: 9.1.7
+version: 9.1.8
```

Comment:
As you are adding new parameters to the chart, please bump the version to a new minor one (9.2.0)

Reply:
Thanks!